Autopilot/boardfiles/Src/main.c

```diff
@@ -148,7 +148,7 @@ int main(void)
   MX_SPI2_Init();
   MX_TIM10_Init();
   MX_TIM11_Init();
-  MX_WWDG_Init();
+  //MX_WWDG_Init();
```

Comment:
If you wanna get rd of the watchdog, gotta do it from the IOC file. Modifying code inside the generated section isn't a good ifdea since it would get written over every time.